### ISSUE TITLE
NPC can't aim with DeathRayWand.

### ISSUE THREAD

**Issue by Ephreaym** [opennox/opennox#648]
NPC will always aim for themselves with the DeathRayWand. Present in NoxReloaded as well.

**Comment by KILAH4716** [comment]
NPCs can't aim period.

IS THAT WHY AI'S ALWAYS MISS?!?!?! They don't aim AT you, just a few pixels in front of themselves?!?!

**Comment by Ephreaym** [comment]
No. The death ray wand is a separate issue. Depending on how they’re instructed they aim at an object or a position.
________________________________
